developing/vcs/master-to-main.md

```diff
@@ -0,0 +1,89 @@
+# [Source Control](./README.md) / Master to Main branch rename
+
+These instructions will help moving repos from using the default branch name `master` and modify it to be
+`main`.
+
+## Instructions to change default branch name
+
+To update the branch name use the npm tool [lgarron/main-branch](https://github.com/lgarron/main-branch):
+
+```sh
+npm install -g main-branch
+main-branch trussworks/master-to-main-test replace master main
+```
+
+Follow up with local modifications:
+
+```sh
+cd git-repo-name
+git pull origin main
+git checkout main
+git branch --delete master
+git branch --set-upstream-to=origin/main main
+git symbolic-ref HEAD refs/heads/main
+git symbolic-ref refs/remotes/origin/HEAD refs/remotes/origin/main
+git fetch --all --prune
+```
+
+Test that the `master` branch has been removed by running this command and getting an error:
+
+```sh
+git checkout master
+```
+
+## Future proofing
+
+Until the `git` tool is modified you'll have to be careful to not creat new repos with `master` as the default
+branch. First make a new git alias:
+
+```sh
+git config --global alias.main '!git init && git checkout -b main'
+git main
+```
+
+You can add an alias to your `bash` or other shell to help:
```

Comment:
can you add something about sourcing that work or reloading the shell tab to take effect? I know that's bitten some juniors and under caffeinated folks (like me) before.